packages/manager-react-components/src/hooks/datagrid/useIcebergV2.tsx

```diff
@@ -1,11 +1,19 @@
 import { useEffect, useState } from 'react';
-import { IcebergFetchParamsV2, fetchIcebergV2 } from '@ovh-ux/manager-core-api';
-import { useInfiniteQuery } from '@tanstack/react-query';
+import {
+  IcebergFetchParamsV2,
+  IcebergFetchResultV2,
+  fetchIcebergV2,
+} from '@ovh-ux/manager-core-api';
+import { Query, useInfiniteQuery } from '@tanstack/react-query';
 import { ColumnSort } from '../../components';
 
 interface IcebergV2Hook {
   queryKey: string[];
   defaultSorting?: ColumnSort;
+  refetchInterval?:
+    | number
+    | false
+    | ((query: Query) => number | false | undefined);
```

Comment:
I think it's good If we can add all query options on params and decompose is on useInfiniteQuery

Reply:
I see what you'd like to do, but it goes beyond this issue, doesn't it?
It may have been intentional not to expose every query option on this hook, which is why I'm only requesting the `refetchInterval` option, the minimum needed here.
However, an issue could certainly be opened to expose every option on icebergV2 or V6 if necessary 👍

Reply:
just for your informations: in my experience using `refreshinterval` with `useInfiniteQuery` without selecting only the actual payload (in `data.pages` with `select` and omitting the page params caused rerenders even if the data didn't changed after refresh (because the cursor changes even tho data is the same) so be careful about that for your use case you might need to use `select` aswell. And it's even more the case with this hook since there is some state update on every `data` change which should be conpletely removed